File: loops-per-second/Loops.java
public class Loops {

  public static void main(String[] args) {
    long before = System.nanoTime();
    long iterations = Long.parseLong(args[0]);
    System.out.println(iterations + " iterations");
    while (iterations > 0) {
      iterations--;
    }
    long time = System.nanoTime() - before;
    System.out.println((time/1000000) + " msecs");
  }

}

File: loops-per-second/loops.clj
; install lein-exec
; lein exec loops.clj 1000000

(defn loops [n]
  (time (loop [i n] (if (> i 0) (recur (dec i))))))

(def iterations
  (read-string (second *command-line-args*)))

(dorun
  (println iterations "iterations")
  (loops iterations))
